bin/run-database.sh

```diff
@@ -108,6 +108,10 @@ bootstrap_configuration () {
 
 if [[ "$1" == "--initialize" ]]; then
     /usr/bin/initialize-certs
+
+    # Start with the default plugins, but persist them on the data volume
+    cp /tmp/enabled_plugins_template "${DATA_DIRECTORY}/enabled_plugins"
```

Comment:
`${DATA_DIRECTORY}/enabled_plugins` == `${RABBITMQ_ENABLED_PLUGINS_FILE}`

```suggestion
    cp /tmp/enabled_plugins_template "${RABBITMQ_ENABLED_PLUGINS_FILE}"
```

Reply:
This was intentional, to avoid defining the location twice. Is there a downside to doing it this way I might have overlooked?

Reply:
Thanks!